extensions/ql-vscode/src/run-remote-query.ts

```diff
@@ -413,3 +429,23 @@ export async function attemptRerun(
     void showAndLogErrorMessage(error);
   }
 }
+
+/**
+ * Updates the default suite of the query pack. This is used to ensure
+ * only the specified query is run.
+ *
+ * @param queryPackDir The directory containing the query pack
+ * @param packRelativePath The relative path to the query pack from the root of the query pack
+ */
+async function fixDefaultSuite(queryPackDir: string, packRelativePath: string): Promise<void> {
+  const packPath = path.join(queryPackDir, 'qlpack.yml');
+  const qlpack = (await yaml.safeLoad(await fs.readFile(packPath, 'utf8'))) as QlPack;
```

Comment:
(Minor question: does the first `await` on this line do anything? I don't think we await `yaml.safeLoad` anywhere else, so just checking I understand 😅 )

Reply:
No...I don't think I need it. I'll remove.